code/game/mecha/mecha.dm

```diff
@@ -1,9 +1,3 @@
-
-
-#define MECHAMOVE_RAND 1
-#define MECHAMOVE_TURN 2
-#define MECHAMOVE_STEP 4

```

Comment:
Already defined in: https://github.com/ParadiseSS13/Paradise/blob/master/code/__DEFINES/mecha.dm